blockstack-sdk/src/main/java/org/blockstack/android/sdk/Network.kt

```diff
@@ -0,0 +1,43 @@
+package org.blockstack.android.sdk
+
+import com.eclipsesource.v8.V8
+import com.eclipsesource.v8.V8Array
+import com.eclipsesource.v8.V8Object
+import org.json.JSONObject
+
+/**
+ * Object giving access to information about the blockstack network.
+ */
+class Network internal constructor(private val v8networkAndroid: V8Object, val v8: V8) {
```

Comment:
What was the thinking behind not just placing these methods in the BlockstackSession object, i.e. `BlockstackSession().getNameInfo(...)`?

Reply:
BlockstackSession is the object that represents a app session. All the transaction methods do not belong to an app/domain. Therefore it is better to have them separated. In the end, the API should provide a builder that does not need a `BlockstackSession` at all if the developer wants to configure the network themselves.

Reply:
Got it.